src/screens/wallet/children/assetsSelectModal.tsx

```diff
@@ -9,13 +9,18 @@ import { CoinBase, CoinData } from '../../../redux/reducers/walletReducer';
 import DEFAULT_ASSETS from '../../../constants/defaultAssets';
 import { setSelectedCoins } from '../../../redux/actions/walletActions';
 import { AssetIcon } from '../../../components/atoms';
+import { profileUpdate } from '../../../providers/hive/dhive';
+import get from 'lodash/get';
+import { updateCurrentAccount } from '../../../redux/actions/accountAction';
 
 export const AssetsSelectModal = forwardRef(({}, ref) => {
   const dispatch = useAppDispatch();
   const intl = useIntl();
 
   const coinsData = useAppSelector((state) => state.wallet.coinsData);
   const selectedCoins: CoinBase[] = useAppSelector((state) => state.wallet.selectedCoins);
+  const pinCode = useAppSelector((state) => state.application.pin);
+  const currentAccount = useAppSelector((state) => state.account.currentAccount);
 
   const [visible, setVisible] = useState(false);
   const [selection, setSelection] = useState(selectedCoins.filter((item) => item.isEngine));
```

Comment:
selection not syncing with updated redux values, for quick fix reset selection using `setSelection()` in `showModal()`

https://user-images.githubusercontent.com/6298342/214293072-f5bdc093-d2bb-419d-9b38-699eb0703d96.mov